tests/rptest/services/redpanda.py

```diff
@@ -574,14 +576,34 @@ def start_node(self,
         if self.coproc_enabled():
             self.start_wasm_engine(node)
 
+        def is_status_ready():
+            status = None
+            try:
+                status = Admin.ready(node).get("status")
+            except:
+                self.logger.error(
+                    f"error on getting status from {node.account.hostname}")
+                e, v = sys.exc_info()[:2]
+                trace = traceback.format_exc()
+                self.logger.error(e)
+                self.logger.error(v)
+                self.logger.error(trace)
+                raise
+            if status != "ready":
+                self.logger.debug(
+                    f"status of {node.account.hostname} isn't ready: {status}")
+                return False
+            return True
+
         def start_rp():
             self.start_redpanda(node)
 
             wait_until(
-                lambda: Admin.ready(node).get("status") == "ready",
+                is_status_ready,
                 timeout_sec=timeout,
+                backoff_sec=1,
                 err_msg=
-                f"Redpanda service {node.account.hostname} failed to start",
+                f"Redpanda service {node.account.hostname} failed to start within {timeout} sec",
                 retry_on_exc=True)
```

Comment:
Remove `retry_on_exc=True` if you don't depend on treating exceptions as `False` now

Reply:
`is_status_ready` still throws exceptions there is `raise` clause in the except clause

Reply:
Ah good point!